### ISSUE TITLE
Not able to locate libnnz11.daylib

### ISSUE THREAD

**Issue by hussainathar** [kubo/fix_oralib_osx#4]
Hi Kubo,  /opt/oracle/instantclient_11_2
I am using **fix_oralib.rb**, after fixing the path, at the time of installing **ruby-oci8** gem I am getting an 
error :  
"" linking shared-object oci8lib_191.bundle
ld: file not found: @rpath/libnnz11.dylib for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make: *** [oci8lib_191.bundle] Error 1 ""

I have the **libnnz11.dylib** file under /opt/oracle/instantclient_11_2/.

In my ** ~/.bash_profile** i have set OCI_DIR path
"

export PATH="$HOME/.rbenv/bin:$PATH"
eval "$(rbenv init -)"
OCI_DIR=/opt/oracle/instantclient_11_2

"

Please help me on this. 
Thank You

**Comment by kubo** [comment]
Could you post more information?
http://www.rubydoc.info/github/kubo/ruby-oci8/file/docs/report-installation-issue.md

> In my ** ~/.bash_profile** i have set OCI_DIR path

You have no need to set OCI_DIR at runtime. It is needed only on installation.

**Comment by kubo** [comment]
@hussainathar Thank you! 

There are two workarounds.

Delete  `-Wl,-flat_namespace` in `rbconfig.rb`, which is in the directory printed by the following command.
```
ruby -r rbconfig -e "puts RbConfig::CONFIG['archdir']"
```

Otherwise, use the absolute path option of fix_oralib.rb
```
cd /opt/oracle/instantclient_11_2
ruby fix_oralib.rb -a
```
(I'll change the default option of fix_oralib.rb to use absolute path.)

I reproduced this issue after I upgraded Xcode from 7.2 to 7.3.
IMO, something was changed between Xcode 7.2 and 7.3.
